### PR TITLE
Modernise the policy wording

### PR DESCRIPTION
**Pull request by aviks**
(no substantive changes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,19 @@ Please note our current policies for accepting entries into METADATA.jl:
...
-   Exceptions may be granted for `julia 0.3` if package authors are willing to vouch that they still test that their packages work on 0.3.
-6. If your package works with Julia 0.4 but not 0.3, then specify `julia 0.4` in your `REQUIRE` file. If the package has had any previous tags which supported `julia 0.3`, then be sure to change the minor or major version number of the package via `Pkg.tag("PkgName", :minor)` for the first tag that no longer supports `julia 0.3`. This makes it possible to create a separate branch for any future bugfix releases that may be needed for the package on Julia 0.3.
-7. We strongly encourage everyone to update METADATA.jl through pull requests, which can be generated for you automatically when using `Pkg.publish()`. GitHub's pull requests allow us to run basic checks on the metadata entries. All developers (especially experienced developers!) are strongly discouraged from editing METADATA.jl directly unless absolutely necessary.
+5. New package version tags must have a minimum Julia version of `0.5` or newer. `0.5-` (0.4 pre-releases) is no longer allowed.
```
Comment (ararslan) [inline README.md]: Should be "0.5 pre-releases" (still says 0.4)

File: README.md [hunk 2]
```diff
@@ -8,19 +8,19 @@ Please note our current policies for accepting entries into METADATA.jl:
...
+   Exceptions may be granted for `julia 0.4` if package authors are willing to vouch that they still test that their packages work on 0.4.
+6. If your package works with Julia 0.6 but not 0.5, then specify `julia 0.6` in your `REQUIRE` file. If the package has had any previous   tags which supported `julia 0.5`, then be sure to change the minor or major version number of the package via `Pkg.tag("PkgName",       :minor)` for the first tag that no longer supports `julia 0.5`. This makes it possible to create a separate branch for any future bugfix releases that may be needed for the package on Julia 0.5.
```
Comment (ararslan) [inline README.md]: Why the insertion of whitespace here?

File: README.md [hunk 3]
```diff
@@ -8,19 +8,19 @@ Please note our current policies for accepting entries into METADATA.jl:
...
+7. We strongly encourage everyone to update METADATA.jl through pull requests, which can be generated for you automatically when you tag a package using Github's UI, provided you have [attobot](https://github.com/integration/attobot) enabled on your repository. Alternatively, you can use the [PkgDev](https://github.com/JuliaLang/PkgDev.jl) package, and its `PkgDev.publish()` function to create PRs. GitHub's pull requests allow us to run basic checks on the metadata entries. All developers (especially experienced developers!) are forbidden from editing METADATA.jl directly unless absolutely necessary in an emergency.
```
Comment (ararslan) [inline README.md]: "Forbidden" seems a little strong, though I don't have a better suggestion offhand.

Comment (aviks) [reply README.md]: `Developers, (especially experienced developers!) should not edit METADATA.jl ...`

?

Comment (ararslan) [reply README.md]: Sure, that seems okay. Though it actually seems a bit unnecessary to single out developers as such. Perhaps just "METADATA.jl should not be edited directory..."?

**Pull request merged**